src/SprykerEco/Yves/Ratepay/Form/ElvSubForm.php

```diff
@@ -38,11 +38,11 @@ public function configureOptions(OptionsResolver $resolver)
     /**
      * @deprecated Use `configureOptions()` instead.
      *
-     * @param \Symfony\Component\OptionsResolver\OptionsResolverInterface $resolver
+     * @param \Symfony\Component\OptionsResolver\OptionsResolver $resolver
```

Comment:
This change looks invalid and possibly BC breaking.

Reply:
`OptionsResolverInterface` doesn't exist anymore

Reply:
Are you sure?
If you look into core code you always see the interface (20x), but never the concrete class being used.

Reply:
Let me check it again

Reply:
Ah ok: The core already uses both, and it seems your way is the one to go :+1:

Reply:
But the deprecated methods should be kept the way they were afaik. See core.